et_replay/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "et_replay"
+version = "1.0.0"
```

Comment:
please start with lower version 0.5.0 ?

Reply:
Updated to 0.5.0

Reply:
Let's keep this matched with Chakra version. Any objections?

Reply:
Updated to 0.0.4

Reply:
Reverted to 0.5.0.